bootstrap/linux-9.0.2.json

```diff
@@ -44,10 +36,18 @@
             "package": "time",
             "version": "1.9.3"
         },
+        {
+            "package": "bytestring",
+            "version": "0.10.12.1"
+        },
         {
             "package": "unix",
             "version": "2.7.2.2"
         },
+        {
+            "package": "binary",
+            "version": "0.8.8.0"
+        },
```

Comment:
Minor comment:  Would it make sense to order this list alphabetically by package name so that diffs are more stable.
In this case, the diff is just a reordering, this could/should be avoided.

Reply:
That's a good idea 👍